packages/core/package.json

```diff
@@ -30,6 +30,6 @@
     "@graphql-codegen/typescript": "^1.23.0",
     "@graphql-codegen/typescript-graphql-request": "^4.5.5",
     "@graphql-codegen/typescript-operations": "^1.18.4",
-    "@nft/api-graphql": "^1.0.0-beta.46.2"
+    "@nft/api-graphql": "^1.0.0-beta.52-prerelease-8"
```

Comment:
Need to wait for non-prerelease version to merge.

Reply:
I let you merge when the library is released